### PR TITLE
Use "numeric-string" type for some MySQLi functions

### PR DESCRIPTION
**Pull request by phansys**
>If the number of rows is greater than PHP_INT_MAX, the number will be returned as a string.

* https://www.php.net/manual/en/mysqli-result.num-rows.php
* https://www.php.net/manual/en/mysqli-stmt.num-rows.php

BTW, is there a way to declare the type for properties like [`mysqli_stmt::$affected_rows`](https://www.php.net/manual/en/mysqli-stmt.affected-rows.php)?

Related to #1116.

### CODE REVIEW

File: resources/functionMap.php [hunk 1]
```diff
@@ -7272,7 +7272,7 @@
 'mysqli_multi_query' => ['bool', 'link'=>'mysqli', 'query'=>'string'],
 'mysqli_next_result' => ['bool', 'link'=>'mysqli'],
 'mysqli_num_fields' => ['int', 'link'=>'mysqli_result'],
-'mysqli_num_rows' => ['int', 'link'=>'mysqli_result'],
+'mysqli_num_rows' => ['int<-1,max>|numeric-string', 'link'=>'mysqli_result'],
```
Comment (ondrejmirtes) [inline resources/functionMap.php]: When it's -1?

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]

**Comment by ondrejmirtes** [comment]
> BTW, is there a way to declare the type for properties

Try to add a stub file, but please add a test for it too (NodeScopeResolverTest), it might be buggy.

File: resources/functionMap.php (hunk 1 above)
Comment (phansys) [reply resources/functionMap.php]: Good catch. I just copied the type from `mysqli_affected_rows`. It's fixed.

**Comment by ondrejmirtes** [comment]
Thank you.

**Pull request merged**